addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/YamahaReceiverBindingConstants.java

```diff
@@ -74,22 +76,51 @@
     public static final String CHANNEL_PLAYBACK_ALBUM = "playback_album";
     public static final String CHANNEL_PLAYBACK_SONG = "playback_song";
 
-    public static final Set<String> CHANNELS_PLAYBACK = Sets.newHashSet(CHANNEL_PLAYBACK_PRESET, CHANNEL_PLAYBACK,
+    public static final Set<String> CHANNELS_PLAYBACK = Sets.newHashSet(CHANNEL_PLAYBACK,
             CHANNEL_PLAYBACK_STATION, CHANNEL_PLAYBACK_ARTIST, CHANNEL_PLAYBACK_ALBUM, CHANNEL_PLAYBACK_SONG);
 
     public static final String UPNP_TYPE = "MediaRenderer";
-
-    public static final CharSequence UPNP_MANUFACTURER = "YAMAHA";
+    public static final String UPNP_MANUFACTURER = "YAMAHA";
 
     public static final String CONFIG_REFRESH = "REFRESH_IN_SEC";
     public static final String CONFIG_HOST_NAME = "HOST";
+    public static final String CONFIG_HOST_PORT = "PORT";
     public static final String CONFIG_ZONE = "ZONE";
     public static final String CONFIG_RELVOLUMECHANGE = "RELVOLUMECHANGE";
 
     public static final String PROPERTY_VERSION = "version";
     public static final String PROPERTY_ASSIGNED_NAME = "assigned_name";
+    public static final String PROPERTY_MENU_ERROR = "menu_error";
+    public static final String PROPERTY_LAST_PARSE_ERROR = "last_parse_error";
 
     public static final String CHANNEL_GROUP_PLAYBACK = "playback_channels";
     public static final String CHANNEL_GROUP_NAVIGATION = "navigation_channels";
     public static final String CHANNEL_GROUP_ZONE = "zone_channels";
+
+    /**
+     * The names of this enum are part of the protocols!
+     * Receivers have different capabilities, some have 2 zones, some up to 4.
+     * Every receiver has a "Main_Zone".
+     */
+    public enum Zone {
+        Main_Zone,
```

Comment:
Can we do this in another PR? This will be changed with the JSON addition, because those names might be XML specific, but I don't know this yet.